options.md

```diff
@@ -577,6 +577,92 @@ This option is compatible only with deployment targets.
       "scale_down": true
     }
 ```
+### feature.db\_branches
```

Comment:
Hm what are these weird `\`?

Reply:
OK, no need to escape everywhere it looks like: https://metalbear.slack.com/archives/C08TYT624SV/p1754908829266439?thread_ts=1754892007.144889&cid=C08TYT624SV